frontApp/app/profile.tsx

```diff
@@ -0,0 +1,221 @@
+import { View, Text, Touchable, TouchableOpacity, TouchableHighlight, Dimensions, StyleSheet, Image, TextInput, ScrollView } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Link, useRouter } from 'expo-router';
+import { Ionicons, MaterialIcons, MaterialCommunityIcons, Entypo } from '@expo/vector-icons';
+
+const Profile = () => {
+    const router = useRouter();
+    const userImage = '../assets/images/GJ-Chillz.jpeg';
+    
+   {/*<Button title="Go back" onPress={() => router.back()} />*/}
+    return (
+        <View style={styles.container}>
+            
+            <View style={styles.header}>
+
+                <View style={styles.header2}>
+                    <View>
+                        <Text style={styles.headerText}>Profile</Text>
+                    </View>
+                </View>
+            </View>
+
+            <TouchableHighlight
+                style = {styles.imageCircle}
+                underlayColor = '#ccc'
+                onPress = { () => alert('Upload an Image!?') } 
+                >
+                <Image source={require(userImage)} resizeMode='cover' style={styles.imageCircle2}>
+                </Image>
+            </TouchableHighlight>
+            <ScrollView>
+                <View>
+                    <Text style={styles.profileGenTitleText}>Account</Text>
+                    <View style={styles.profileView}>
+                        <View style={styles.iconSettings}>
+                            <MaterialIcons name="date-range" size={32} color="black" />
+                        </View>
+                    <TextInput style={styles.accountText} value = "Date of Birth" editable = {false}/>
```

Comment:
With these being 
`editable = {false}` 
Do we want to make a function that tracks the state and use the edit profile button to toggle it on and off or something like that?

Reply:
So yeah that was the idea, the edit button would toggle this and editable field (true/false) and it would allow the user to change these values. It should also add a new password/password confirm field if the user choose to update their password